### PR TITLE
Add Admin CreatePartitions API call

### PR DESCRIPTION
**Pull request by alexef**
I've been using it for some time in a personal project.

Protocol should match:https://github.com/apache/kafka/blob/1.0/clients/src/main/java/org/apache/kafka/common/requests/CreatePartitionsRequest.java and  https://github.com/apache/kafka/blob/1.0/clients/src/main/java/org/apache/kafka/common/requests/CreatePartitionsResponse.java

### CODE REVIEW

File: kafka/protocol/__init__.py [hunk 1]
```diff
@@ -42,4 +42,6 @@
     31: 'DeleteAcls',
     32: 'DescribeConfigs',
     33: 'AlterConfigs',
+    36: 'SaslAuthenticate',
```
Comment (alexef) [inline kafka/protocol/__init__.py]: I saw it was missing, but anyway where is API_KEYS used?

Comment (tvoinarovskyi) [reply kafka/protocol/__init__.py]: It's not used atm.

File: kafka/protocol/admin.py [hunk 2]
```diff
@@ -333,3 +332,33 @@ class SaslAuthenticateRequest_v0(Request):
 
 SaslAuthenticateRequest = [SaslAuthenticateRequest_v0]
 SaslAuthenticateResponse = [SaslAuthenticateResponse_v0]
+
+
+class CreatePartitionsResponse_v0(Response):
+    API_KEY = 37
+    API_VERSION = 0
+    SCHEMA = Schema(
+        ('topic_error_codes', Array(
```
Comment (tvoinarovskyi) [inline kafka/protocol/admin.py]: Seems like you missed `throttle_time_ms` field. Lets set the name to `topic_errors` to be exact to docs also. See https://kafka.apache.org/protocol#The_Messages_CreatePartitions.

Comment (alexef) [reply kafka/protocol/admin.py]: Ouch, my bad, pushed a fix.

**Comment by alexef** [comment]
@tvoinarovskyi thank you for the review! Is there something else I need to do to get this merged, or just wait for more approval?

**Comment by jeffwidman** [comment]
Looked good to me, so I merged it. Thanks!

File: kafka/protocol/admin.py (hunk 2 above)
Comment (Vijay-P) [reply kafka/protocol/admin.py]: Is Line 335 in kafka/admin/client.py a residual mistake from this change?

**Pull request merged**